tests/manage/z_cluster/cluster_expansion/test_osd_topology_spread.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from _collections import defaultdict
+
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.node import get_ocs_nodes, get_osds_per_node, get_node_name
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_ocs_version,
+    skipif_ocp_version,
+    tier1,
+)
+
+
+@tier1
+@skipif_ocs_version("<4.7")
+@skipif_ocp_version("<4.6")
+@pytest.mark.last
+class TestOsdTopologySpread(ManageTest):
+    def test_osd_topology_spread(self):
+        """
+        Test to verfify for even distribution of OSDs
+        """
+        topology_key = helpers.get_failure_domin()
+        if topology_key == "zone":
+            failure_domain_label = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
we also need to use a new zone label which is "topology.kubernetes.io/zone=a"

Reply:
Why it is required to check for the value (i.e `a` )in the label?

Reply:
what if the user does not add a failure-domain label here test will fail as failure-domain.beta.kubernetes.io/zone is deprecated in favor of topology.kubernetes.io/zone.

Reply:
i have just added an example here (ie `a`)

Reply:
ah i see, `failure_domain_label` holds just the label name and its value is extracted in line 36
`failure_domain = labels.get(failure_domain_label)`

Reply:
As discussed, used stable label `topology.kubernetes.io/zone`